### PR TITLE
Work Section

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
- Add the work section mobile view
- I fixed linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,9 +44,55 @@ <h2><a href="#">LET'S CONNECT</a></h2>
...
+                        <li><a href="#"><img  src="./images/angellist.svg" alt="angellist"></a></li>
+                       
+                    </ul>
+                </div>
+                    <img src="./images/Enabled.svg" alt="">
+                
+                <img src="./images/Frame11.svg" alt="">
+                <img src="./images/Frame9.svg" alt="">
+                <img src="./images/Frame10.svg" alt="">
+
```
Comment (ahmetbozaci) [inline index.html:85]: - [ ] After main section you should add works part wrapped by `section` tag. You don't have this section.

- [ ] Original design has 4 works. Please add all of them.

image
____
_Note for next TSE. Please check the work section requirements._

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 

## Hello @Isabellanamutenda ✋🏼  Your project is great. You are almost done. However there are some issues you should fix. Please fix them then your project will be approved.


### ♻️ Request changes ♻️
- [ ] You should update your README file. Please use this [template](https://github.com/microverseinc/readme-template) and make sure there are no  broken links and empty sections. If you won't use some sections, delete them.

_**Check the comment about that  under the review.**_








### OPTIONAL SUGGESTIONS

#### These are optional 

- [x] Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`. Please watch this [video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) and check this [article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
- [x]  Instead of using pictures for icons you can use icons from [font-awesome](https://fontawesome.com/start). 

- [x] For DRY css you can  combine duplicate properties. You can create class `.d-flex { display: flex}` and use this class in html file.



- For more information about DRY principles you can check this [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 


### Cheers and Happy coding! 💻 👏
cat on laptop
___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

File: index.html (hunk 1 above)
Comment (Isabellanamutenda) [reply index.html]: I have added all of them

File: index.html [hunk 2]
```diff
@@ -2,50 +2,71 @@
...
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <div class="head">
```
Comment (rloterh) [inline index.html:13]: - [ ] OPTIONAL
Your header/ navbar content is not showing. This is the main entry point of your page. Kindly ensure to fix this aspect.

File: index.html [hunk 3]
```diff
@@ -43,9 +44,55 @@ <h2><a href="#">LET'S CONNECT</a></h2>
...
+                <img src="./images/Frame9.svg" alt="">
+                <img src="./images/Frame10.svg" alt="">
+
+               <!-- <h5>Languages</h5>
+               <h5>Frameworks</h5>
+               <h5>Skills</h5> -->
+               
+
+            
+
```
[the remark this answers is not in the document]
Comment (rloterh) [reply index.html]: Well resolved

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: Well resolved

File: style.css [hunk 4]
```diff
@@ -8,78 +8,299 @@ body {
...
+  background-image: url("images/Card\ Works (1).svg");
+  background-repeat: no-repeat;
+  background-size: 495px 688px;
+  position: absolute;
+  width: 100%;
+  height: 687px;
+  background-color: #e5e5e5;
+  top: 882px;
+}
+
```
Comment (rloterh) [inline style.css:42]: - [ ] OPTIONAL
Stylelint is detecting errors in your code.

image

Kindly ensure to improve upon this aspect

**Review by rloterh — APPROVED** [review]
Hi Ashley, 

Great job for making the necessary changes recommended by the previous reviewer. However, there's always room for improvement.
Kindly consider my OPTIONAL comments

## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

- -Also remember to use a descriptive ReadMe for your project. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.-

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**

<br>
Your project is complete! Congratulations! 🥂There is nothing else to say other than... it's time to merge it 👍.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Isabellanamutenda) [reply style.css:42]: Fixed

**Pull request merged**